core/src/avm2/globals/string.rs

```diff
@@ -564,12 +646,16 @@ pub fn create_class<'gc>(mc: MutationContext<'gc, '_>) -> GcCell<'gc, Class<'gc>
         ("concat", concat),
         ("indexOf", index_of),
         ("lastIndexOf", last_index_of),
+        ("localeCompare", locale_compare),
         ("match", match_s),
         ("replace", replace),
+        ("search", search),
         ("slice", slice),
         ("split", split),
         ("substr", substr),
         ("substring", substring),
+        ("toLocaleLowerCase", to_locale_lower_case),
```

Comment:
alternatively, this can skip a new method entirely and just do
```
"toLocaleLowerCase", to_lower_Case),
```

Reply:
Done